scripts/calculateRevenue/protocols/beefy.ts

```diff
@@ -1,11 +1,100 @@
-import { RevenueResult } from '../../types'
+import { RevenueResult, NetworkId } from '../../types'
 import { fetchWithTimeout } from '../../utils/fetchWithTimeout'
-
+import { getViemPublicClient, getStrategyContract } from './utils/viem'
+import { Address } from 'viem'
 export type BeefyVaultTvlData = [string, number]
 
 const BEEFY_API_URL = 'https://databarn.beefy.com/api/v1/beefy'
+const DEFI_LLAMA_API_URL = 'https://coins.llama.fi'
+
 const ONE_WEEK = 7 * 24 * 60 * 60 * 1000
 
+const NETWORK_ID_TO_DEFI_LLAMA_CHAIN: Partial<{
+  [networkId in NetworkId]: string // eslint-disable-line @typescript-eslint/no-unused-vars
+}> = {
+  [NetworkId['ethereum-mainnet']]: 'Ethereum',
+  [NetworkId['arbitrum-one']]: 'Arbitrum',
+  [NetworkId['op-mainnet']]: 'Optimism',
+  [NetworkId['celo-mainnet']]: 'Celo',
+  [NetworkId['polygon-pos-mainnet']]: 'Polygon',
+  [NetworkId['base-mainnet']]: 'base',
```

Comment:
interesting that this is lowercase

Reply:
Actually, I think these can all be lowercase. The endpoint accepts certain chains as both uppercase/lowercase, but base only worked when lowercase. After some testing, it looks like all chains work when lowercased, so I'll just do that for them all.